test/checkers/trace_checker.c

```diff
@@ -131,13 +132,20 @@ iter_pointer_value(struct entry_it it)
     return coldtrace_entry_parse_ptr(it.buf);
 }
 
+uint64_t
+iter_size(struct entry_it it)
+{
+    return coldtrace_entry_size(it.buf);
+}
+
 static void
 init_expected_entry_iterator(struct next_expected_entry_iterator *iter,
                              struct expected_entry *exp)
 {
     iter->atleast = exp->atleast;
     iter->atmost  = exp->atmost;
     iter->check   = exp->check;
+    iter->size    = exp->size;
```

Comment:
If I remember correctly, we have atleast and atmost because we change these values. Size doesn't change, do we need it here for another reason?